### PR TITLE
Add script to check case when SDL.GetUserFriendlyMessage request come…

### PR DESCRIPTION
**Pull request by vvvakulenko**
…s with language parameter

### CODE REVIEW

**Comment by vvvakulenko** [comment]
@maria-mih Please review

File: test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+-- Requirement summary:
+-- [Policies]: SDL.GetUserFriendlyMessage ("language")
+-- [HMI API] SDL.GetUserFriendlyMessage request/response
+--
+-- Description:
+-- 1. Precondition: SDL is started, HMI is started, App is registered
+-- 2. Steps: Activate App, in SDL.GetUserFriendlyMessage parameter "language" should be present
+--
+-- Expected result:
+-- PoliciesManager must respond with the information from the sub-section with the name correspondent to "language" value
```
Comment (maria-mih) [inline test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua]: Please write what is the expected result in terms of communication.
You can use verification criteria from requirement.

File: test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+---------------------------------------------------------------------------------------------
+ 
+--[[ General configuration parameters ]]
+Test = require('connecttest')
+require('cardinalities')
```
Comment (maria-mih) [inline test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua]: Please move to section --[[General Settings for configuration]]

File: test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+require('mobile_session')
```
Comment (maria-mih) [inline test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua]: Please move to section --[[General Settings for configuration]]

File: test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+-- Description:
+-- 1. Precondition: SDL is started, HMI is started, App is registered
+-- 2. Steps: Activate App, in SDL.GetUserFriendlyMessage parameter "language" should be present
+--
+-- Expected result:
+-- PoliciesManager must respond with the information from the sub-section with the name correspondent to "language" value
+---------------------------------------------------------------------------------------------
+ 
+--[[ General configuration parameters ]]
+Test = require('connecttest')
```
Comment (maria-mih) [inline test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua]: Please move to section --[[General Settings for configuration]]

File: test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+commonSteps:DeleteLogsFileAndPolicyTable()
+ 
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeleteLogsFileAndPolicyTable()
+ 
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:ActivateApp_with_language()
```
Comment (maria-mih) [inline test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua]: Please add to name of functions in this section TestStep_ e.g.
function Test:TestStep_ActivateApp_with_Language_Message

Comment (vvvakulenko) [inline test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua]: @maria-mih done in commit

File: test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua (hunk 3 above)
Comment (vvvakulenko) [inline test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua]: @maria-mih done in commit 414f68b

File: test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua (hunk 2 above)
Comment (vvvakulenko) [inline test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua]: @maria-mih done in commit 414f68b

File: test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua (hunk 4 above)
Comment (vvvakulenko) [inline test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua]: @maria-mih done in commit 414f68b

File: test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua (hunk 1 above)
Comment (vvvakulenko) [inline test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua]: @maria-mih done in commit 414f68b

**Comment by vvvakulenko** [comment]
@maria-mih Please check updates

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**